### ISSUE TITLE
Map Screen Blacked Out

### ISSUE THREAD

**Issue by TheMurderUnicorn** [CleverRaven/Cataclysm-DDA#49590]
# Describe the bug
Decent portion of the map screen is simply blacked out.  Screenshots below.  Someone on Discord said it's a resolution issue, couldn't really give more information and I didn't see an issue for it so I thought I'd make one.

# Steps To Reproduce
I'm not sure if it's reproducable because I'm not sure what's causing it.  For me it just is.  If it's truly connected to resolution my settings are this: 
![image](https://user-images.githubusercontent.com/41651793/124402759-44dced80-dd00-11eb-8195-e046b8e36160.png)
![image](https://user-images.githubusercontent.com/41651793/124402768-50c8af80-dd00-11eb-9036-54d81d8e8ddc.png)

# Expected behavior
I would expect the map to be displayed in the entire map panel rather than having maybe 20-25% blacked out.

# Screenshots
![image](https://user-images.githubusercontent.com/41651793/124402785-7786e600-dd00-11eb-869e-ae7e5e22f0a5.png)
![image](https://user-images.githubusercontent.com/41651793/124402795-853c6b80-dd00-11eb-8cb1-320a20676349.png)

# Versions and configuration
- OS: Windows
    - OS Version: 10.0 2009
- Game Version: ea0cd21 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Tutorial Mod [tutorial_mod]
]

# Additional context
The modlist above was just from the random save I took these pictures on.  the issue persists across all worlds, can't imagine my mods have anything to do with it but figured I'd mention it.

You can also see that town nameplates are displaying incorrectly but I'm assuming that's a different issue for some other bug/problem.

On a more personal note, it took me a really long time to dial in my settings to a point I liked them.  I know many of you know I do YouTube and scale things up to make things more visible for mobile users.  Tweaking a lot of these settings has always been finicky, at least for my smooth brain, and I'm hoping I don't have to play with default settings for this one issue.  I have no idea what would be causing it, hopefully ya'll do.

**Comment by TheMurderUnicorn** [comment]
I believe this has been resolved but I'm guessing no one noticed this issue and didn't mark it as resolved.  So I will close this.  If it's temporary and something random happened I'll reopen or make a new issue or whatever.
